calcom/cal.com
# fix: random multiple duration value input [CAL-887]

## What does this PR do?

New booker allowed any duration to be passed by query parameter to override event type duration, even a non-multiple-duration event-type.

Fixes #6433 

## Type of change


## How should this be tested?

- Go to `/pro/30min` event-type and add `?duration=45` to the URL, which will be deleted and ignored now. 
- Go to `/pro/multiple-duration` event-type and modify the 75 default value for the `duration` query parameter to an allowed value and see it go through, and modify it to a non-allowed value and see it change to the default value 75

## Mandatory Tasks
